src/css/fonts.css

```diff
@@ -0,0 +1,59 @@
+/**
+Logo fonts
+**/
+@font-face {
+  font-family: 'icomoon';
+  src:  url('fonts/icomoon.eot?yl49be');
+  src:  url('fonts/icomoon.eot?yl49be#iefix') format('embedded-opentype'),
+  url('fonts/icomoon.ttf?yl49be') format('truetype'),
+  url('fonts/icomoon.woff?yl49be') format('woff'),
+  url('fonts/icomoon.svg?yl49be#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+/**
+Text fonts
+**/
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap');
```

Comment:
I'm not sure why, but my IDE tells me this should be at the top of the file

Reply:
I think your IDE is right as per [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@import). I'll correct this.